otx/algorithms/classification/configs/configuration.yaml

```diff
@@ -305,6 +305,7 @@ algo_backend:
     header: train type
     options:
       INCREMENTAL: "INCREMENTAL"
+      SUPERVISEDCONTRASTIVE: "SUPERVISEDCONTRASTIVE"
```

Comment:
As Songki said, we would like to recommend to add supcon option in configuration.yaml like tiling method in detection